src/StakeRegistry.sol

```diff
@@ -418,6 +418,7 @@ contract StakeRegistry is StakeRegistryStorage {
                 "StakeRegistry._addStrategyParams: cannot add strategy with zero weight"
             );
             strategyParams[quorumNumber].push(_strategyParams[i]);
+            strategiesPerQuorum[quorumNumber].push(_strategyParams[i].strategy);
```

Comment:
Secondary note - need to ensure strategies are removed from this list in `removeStrategies`!

Reply:
oh nvm i see jeffrey is saying that above